### ISSUE TITLE
Store the pk folder data inside ROMFS

### ISSUE THREAD

**Issue by DanTheMan827** [gocario/PHBank#31]
I think it would make more sense to store the template data, move list, etc inside the ROMFS rather than have the user extract it to the SD card...

You could also store the blank bank file there and copy it to the SD card if it doesn't exist.

**Comment by gocario** [comment]
For the most of the files, I try to load from sdmc:/ then romfs:/
(ie: https://github.com/gocario/PHBank/blob/master/source/data_manager.cpp#L141-L150)
But something wasn't working, no idea why, I didn't worked on it that much afterward.

I don't want to place the bank file into the SaveArchive tho, because it has to be shared between the .cia and the .3dsx version. (The blank bank file is just a file filled with zeroes)

**Comment by DanTheMan827** [comment]
Ah, yeah... I wasn't saying store the bank file in SaveArchive...

You seem to have a few folders inside the pk folder in the rar archive that look to be localization files that I was talking about... is it necessary to put those on the SD card anymore?

**Comment by gocario** [comment]
Yeah, the romfs problem is still here, I haven't put my head in for a while, and I have other things to do.
